src/picker_gui.cpp

```diff
@@ -678,3 +678,13 @@ std::unique_ptr<NWidgetBase> MakePickerTypeWidgets()
 
 	return MakeNWidgets(picker_type_widgets, nullptr);
 }
+
+void InvalidateAllPickerWindows()
+{
+	InvalidateWindowClassesData(WC_BUS_STATION, PickerWindow::PICKER_INVALIDATION_ALL.base());
+	InvalidateWindowClassesData(WC_TRUCK_STATION, PickerWindow::PICKER_INVALIDATION_ALL.base());
+	InvalidateWindowClassesData(WC_SELECT_STATION, PickerWindow::PICKER_INVALIDATION_ALL.base());
+	InvalidateWindowClassesData(WC_BUILD_WAYPOINT, PickerWindow::PICKER_INVALIDATION_ALL.base());
+	InvalidateWindowClassesData(WC_BUILD_OBJECT, PickerWindow::PICKER_INVALIDATION_ALL.base());
+	InvalidateWindowClassesData(WC_BUILD_HOUSE, PickerWindow::PICKER_INVALIDATION_ALL.base());
```

Comment:
Isn't the `EnumBitSet` `ConvertibleThroughBase`? Then the `.base()`s shouldn't be needed.